src/engines/kotor/gui/widgets/kotorwidget.cpp

```diff
@@ -74,12 +74,16 @@ void KotORWidget::show() {
 		_quad->show();
 	if (_text)
 		_text->show();
+    if (_border)
+        _border->show();
 }
 
 void KotORWidget::hide() {
 	if (isInvisible())
 		return;
 
+    if (_border)
```

Comment:
About the Indentation, I recognized, that my IDE inserted some tabs while i was not looking. I will definitely correct those.